frontend/src/components/Elections/ElectionInvitations.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react'
+import { useGetElections } from "../../hooks/useAPI";
+import ElectionsTable from './ElectionsTable';
+
+export default () => {
+
+    const { data, isPending, error, makeRequest: fetchElections } = useGetElections();
+
+    useEffect(() => {fetchElections()}, []);
+
+    const electionInvitations = React.useMemo(
+        () => data && data.elections_as_voter ? data.elections_as_voter : [],
```

Comment:
Just in case you haven't seen it and are into it you can equivalently write `data && data.elections_as_voter ? data.elections_as_voter : []` as `data?.elections_as_voter ?? []`

More in [release notes from ts 3.7](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html)

(Obviously the existing way is also fine/equivalent)

Reply:
I do like that, I'll update the PR